### ISSUE TITLE
Adjust storageCapcity didn't work

### ISSUE THREAD

**Issue by sinkan851014** [matsim-org/matsim-code-examples#726]
Hi MATSim community, 

According to MATSim book , if the sample data = 1% the flow capacity should be set to 0.01.
But it may cause problem like traffic would stuck in network.

Also found this issue in my scenario, so I set storageCapcity for test.

However, no matter how I set storageCapcity , the results were same.

Maybe there are something else or code should be modified?
Thanks!

[logfile.log](https://github.com/matsim-org/matsim-code-examples/files/8692981/logfile.log)

[config.xml.zip](https://github.com/matsim-org/matsim-code-examples/files/8692982/config.xml.zip)

**Comment by Janekdererste** [comment]
For 1% you would have to set the storage and flow capacity to 0.01. 

0.1 would be a good fit for a 10% sample size

**Comment by sinkan851014** [comment]
@Janekdererste 
Sorry! I mistyped the value of flow capacity in article.

I actually set 	`<param name="flowCapacityFactor" value="0.01" />` in my config file.
But whatever I set `	<param name="storageCapacityFactor" value="" />`  to even 1 or 0.01, the output always same.

Maybe this setting didn't work?

**Comment by Janekdererste** [comment]
Alright, that is indeed the correct factor for your sample size. 

What exactly do you mean by "the output is always the same"? Do you expect different congestion patterns than what you are observing?

Propbably it will help if I take a look at your scenario. For this I would need your input files. Preferrably packed as zip file or similar.

**Comment by sinkan851014** [comment]
@Janekdererste
Yes, I would expect different congestion patterns in two differnet setting scenario. However the traffic was same in visualization by Via. 

My plan input data was only 0.16% sample from survey data . Now I still try to find method expanding it, in this data simplely added same plan 10 times to test the flowCapacity and storageCapacity setting. And the code is from [matsim-example-project](https://github.com/matsim-org/matsim-example-project), running by RunMATSim class.

Thanks!
[input data.zip](https://github.com/matsim-org/matsim-code-examples/files/8711048/input.data.zip)
